src/structures/paging/mapper/mod.rs

```diff
@@ -53,20 +59,26 @@ pub enum TranslateResult {
     Frame4KiB {
         /// The mapped frame.
         frame: PhysFrame<Size4KiB>,
+        /// Flags in the pagetable for this entry
```

Comment:
Above we use the order (frame, offset, flags), the declaration here should be consistent. Either we should always use (frame, offset, flags) or (frame, flags, offset).

Reply:
I've reordered the fields to adhere to (frame, offset, flags), which is think is the neater way to add this field.